x/leverage/simulation/genesis.go

```diff
@@ -0,0 +1,92 @@
+package simulation
+
+import (
+	"encoding/json"
+	"fmt"
+	"math/rand"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/types/module"
+	"github.com/umee-network/umee/x/leverage/types"
+)
+
+// Simulation parameter constants
+const (
+	interestEpochKey                = "interest_epoch"
+	completeLiquidationThresholdKey = "complete_liquidation_threshold"
+	minimumCloseFactorKey           = "minimum_close_factor"
+	oracleRewardFactorKey           = "oracle_reward_factor"
+)
+
+// GenInterestEpoch produces a randomized InterestEpoch in the range of [10, 100]
+func GenInterestEpoch(r *rand.Rand) int64 {
+	return int64(10 + r.Intn(100))
```

Comment:
```suggestion
	return int64(10 + r.Intn(90))
```
For range to end at 100